pure.zsh

```diff
@@ -175,7 +175,11 @@ prompt_pure_preprompt_render() {
 		$cleaned_ps1
 	)
 
-	PROMPT="${(j..)ps1}"
+    PROMPT=""
+    if ((${(M)#jobstates:#suspended:*} != 0)); then
+      PROMPT+='%F{$prompt_pure_colors[git:dirty]}✦ '
```

Comment:
This need a new color instead of reusing one.

Reply:
This should be moved higher up so that it's part of the `preprompt_parts` array (instead of doing `PROMPT` manipulation here).